server/src/repositories/storage.repository.ts

```diff
@@ -214,7 +214,7 @@ export class StorageRepository implements IStorageRepository {
   }
 
   private asGlob(pathToCrawl: string): string {
```

Comment:
why is there a square bracket?

Reply:
Around the special characters? Turns out, that's how you escape double quotes, single quotes and asterisks when using a dynamic glob pattern. A literal " is written as ["]